examples/menubar/menubar-1/js/MenubarItemLinks.js

```diff
@@ -138,6 +142,10 @@ MenubarItem.prototype.handleKeydown = function (event) {
       break;
 
     default:
+      if (isPrintableCharacter(char)) {
+        this.menubar.setFocusByFirstCharacter(this, char);
+        flag = true;
+      }    
```

Comment:
There is disallowed trailing white space and other JSCS errors. Please run `npm run jscs-fix`